docs/markdown/Introduction/welcome-to-pants.md

```diff
@@ -16,7 +16,7 @@ Pants installs, orchestrates and runs dozens of standard underlying tools - comp
 
 Pants is designed to be easy to adopt, use, and extend. It doesn't require you to refactor your codebase or to create and maintain massive amounts of build metadata. You invoke it directly on source files and directories, so it doesn't require users to adopt a new conceptual model.
 
-Pants is currently focused on Python, Go, Java, Scala, Shell, and Docker, with more languages and frameworks coming soon. [The Pants community](doc:the-pants-community) is friendly and helpful, and supported by [Toolchain](https://toolchain.com/), a venture-backed company whose mission is to enable fast, stable, ergonomic developer workflows for everyone.
+Pants is currently focused on Python, Go, Java, Scala, Shell, and Docker, with more languages and frameworks coming soon. [The Pants community](doc:the-pants-community) is friendly and helpful.
```

Comment:
After removing the toolchain part, the sentence feels a tad too short. Perhaps adding a short note about the community would make the sentence more complete.

```suggestion
Pants is currently focused on Python, Go, Java, Scala, Shell, and Docker, with more languages and frameworks coming soon. [The Pants community](doc:the-pants-community) is friendly and helpful and welcomes involvement of anyone who is interested in creating modern software development tooling.
```

Reply:
@tnielens do you find this suggestion sensible? :)

Reply:
Absolutely :). Sorry for the late reaction.